packages/docz-theme-default/src/components/ui/Render/index.tsx

```diff
@@ -388,13 +388,13 @@ export class Render extends Component<RenderComponentProps, RenderState> {
 
   private transformCode(code: string): string {
     return `
-      const App = ({ children }) => (
+      const _App = ({ children }) => (
```

Comment:
```suggestion
      const _Docz_App = ({ children }) => (
```

What does everyone think about namespacing this so that it is even less likely to collide in the future? Since the user shouldn't need to interact with this directly I can't think of too many negatives to doing this.